lib/module.js

```diff
@@ -103,18 +106,10 @@ function Typo3 (options) {
     }
 
     this.nuxt.hook('components:dirs', (dirs) => {
-      // TODO: use ignore, try with '!(T3Form/**|T3CeFormFormframework/**|T3CeFeloginLogin/**)' dont work. https://github.com/isaacs/node-glob#glob-primer
       dirs.push({
-        path: join(__dirname, 'components/T3Form'),
-        pattern: 'T3Form.vue',
-        level: 999
-      }, {
-        path: join(__dirname, 'components/T3CeFormFormframework'),
-        ignore: ['index.js'],
-        level: 999
-      }, {
-        path: join(__dirname, 'components/T3CeFeloginLogin'),
-        ignore: ['index.js'],
+        path: join(__dirname, 'components'),
```

Comment:
@kubilaymelnikov does it work for you ? because now I see only the form components on server side source, but on the client side nuxt says they are not available